picasso/src/main/java/com/squareup/picasso3/Request.java

```diff
@@ -101,8 +101,7 @@
   @Nullable
   public final Bitmap.Config config;
   /** The priority of this request. */
-  @Nullable
-  public final Priority priority;
+  final Priority priority;
```

Comment:
Lost public. Intentional?

Reply:
While removing the annotation, I noticed the IDE inspection highlighted it as not requiring a public access modifier, so I removed it.  

It also fails a Lint check when `public` with the following error: 
```
/Users/jrod/Development/picasso/picasso/src/main/java/com/squareup/picasso3/Request.java:104: Error: Unknown nullability; explicitly declare as @Nullable or @NonNull to improve Kotlin interoperability; see https://android.github.io/kotlin-guides/interop.html#nullability-annotations [UnknownNullness]
  public final Priority priority;
               ~~~~~~~~
```

I'll add the `@NonNull` annotation instead.